### PR TITLE
Matches Component

### PR DESCRIPTION
**Pull request by BrandonAlexanderLopez**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
The matches component has been added.
It displays the company name, symbol, and price.

* **What is the current behavior?What does the code do before you made the new changes?** 
Currently the companies and stocks are hard-coded meaning the values are static.
Once we incorporate the API's and add more functionality to our app then the matches will be dynamic

* **What is the new behavior?**
The matches component has been added.
It displays the company name, symbol, and price.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)* 
* NO, nothing breaks

* **Other information**:
CURRENT OUTPUT:
image

### CODE REVIEW

**Review by Sakamoto448 — DISMISSED** [review]
Looks good to me. Ran the code and saw everything from the picture that you had. Nothing crashed and all static values were present. Nice Job!

File: src/components/Matches.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Table } from 'reactstrap';
+
+const Matches = (props) => {
```
Comment (Codubee) [inline src/components/Matches.js]: I see that the components you create start with the const keyword. Can you change these into functional components instead?

**Review by Codubee — CHANGES_REQUESTED** [review]
Left some comments

**Review by Codubee — CHANGES_REQUESTED** [review]
Great first attempt here Brandon, but can you display a javascript array of objects here instead of hard coded values. I included some documentation in the ticket description if youd like some help.

**Review by BrandonAlexanderLopez — COMMENTED** [review]
That has been reviewed and changed.

**Comment by BrandonAlexanderLopez** [comment]
image
The matches are now an array of objects however the output doesn't look as nice as if it was in the form of a table.

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**